modules/stochastic_tools/doc/content/modules/stochastic_tools/examples/bayesian_uq_1d_diff.md

```diff
@@ -0,0 +1,207 @@
+    id=infer-var
+    caption=Complete input file for inferring both model parameters and the variance term using the differential evolution sampler.
+
+## Result and Analysis
+
+The output `.json` file contains samples across different processors at different timesteps. Practically, all MCMC samples are generated with serial auto-correlations. Therefore, it is essential to diagnoze the sample auto-correlation for predictable UQ quality. Two primary diagnostics are presented here for analyzing the sample quality: the integrated auto-correlation time ($\tau_p$), and the effective sample size (ESS) [!cite](dhulipala2023massively). $\tau_p$ describes the interval after which a sample can be considered independent given a current sample index, and ESS describes the number of "effective" samples after taking into consideration the sample auto-correlation. To obtain "independent" samples from the Markov chains, the first 100 samples are discarded for "burn-in", with the remaining samples thinned by $\tau_p/2$. The ESS of the generated samples are calculated based on the thinned samples. As presented in [metrics_case1], $\tau_p$ for IMH, SS and DES decreases, and ESS for the three algorithms increases corresponding, indicating the best sampling qualify by DES, followed by SS, then IMH. 
+
+!table id=metrics_case1
+        caption=Integrated autocorrelation time ($\tau_p$) and effective sample size (ESS) when inferring model parameters
+| Algorithm              | $\tau_p$                 | ESS                  |
+|:-----------------------|:-------------------------|:---------------------|
+| IMH                    | 123.908                  | 9.528                |
+| SS                     | 41.779                   | 46.719               |
+| DES                    | 11.217                   | 676.547              |
+
+When inferring only the boundary conditions, all three samplers are applied, with 1,000 serial steps simulated with 5 parallel proposals per step.
+[scatter_plots_case1] shows the posterior distributions of the calibration parameters $u_{left}$ and $u_{right}$. A strong linearatiy is observed for the two parameters as would be expected from the underlying physics. The IMH samples appear to be extremely localized compared to DES. The SS samples are almost as good as samples generated by DES in terms of exploring the parameter space. The DES samples are the best from this aspect. More detailed posterior distributions are presented in [case1_inference_results] for all three algorithms IMH, SS and DES. It is generally observed that the posteriors using IMH are restricted in the parameter space. Whereas, the posteriors for both SS and DES are spread across the parameter space and look comparable to each other. This can be observed by comparing [case1_inference_results]b and [case1_inference_results]c for the posterior of $u_{left}$, and [case1_inference_results]e and [case1_inference_results]f for the posterior of $u_{right}$. 
```

Comment:
```suggestion
[scatter_plots_case1] shows the posterior distributions of the calibration parameters $u_{left}$ and $u_{right}$. A strong linearity is observed for the two parameters as would be expected from the underlying physics. The IMH samples appear to be extremely localized compared to DES. The SS samples are almost as good as samples generated by DES in terms of exploring the parameter space. The DES samples are the best from this aspect. More detailed posterior distributions are presented in [case1_inference_results] for all three algorithms IMH, SS and DES. It is generally observed that the posteriors using IMH are restricted in the parameter space. Whereas, the posteriors for both SS and DES are spread across the parameter space and look comparable to each other. This can be observed by comparing [case1_inference_results]b and [case1_inference_results]c for the posterior of $u_{\mathrm{left}}$, and [case1_inference_results]e and [case1_inference_results]f for the posterior of $u_{\mathrm{right}}$. 
```